addons/editor/functions/fnc_initDisplayCurator.sqf

```diff
@@ -73,6 +69,25 @@ _display displayAddEventHandler ["KeyDown", {call FUNC(handleKeyDown)}];
     }];
 } forEach IDCS_SIDE_BUTTONS;
 
+// Fix selecting a path in group trees preventing selecting an editable icon using left click
+// This matches the behaviour of other trees by deselecting any non-placement path when selecting an editable icon
+{
+    private _ctrl = _display displayCtrl _x;
+    _ctrl ctrlAddEventHandler ["MouseButtonDown", {
+        if (RscDisplayCurator_sections select 0 == 1) then {
```

Comment:
Wasn’t there a function or global var already for this as a shortcut?

Reply:
If you mean for: `RscDisplayCurator_sections select 0` - no, not that I know of. This is the only way to get the current mode.